.github/workflows/periodic.yml

```diff
@@ -1,13 +1,19 @@
 ---
-name: Markdown Link Checks
+name: Periodic
+
 on:
   schedule:
     - cron: "0 0 * * 0"
+  push:
+    branches:
+      - master
+    tags:
+      - 'v**'
```

Comment:
Why would periodic run on push?
I don't think this makes sense, and looks like it could even confuse

Reply:
Right, that part is confusing I agree, for the push it's not periodic anymore. So it would show as "Periodic" on a push.

Could it be part of Validation and that's it?

Reply:
I think we have Validation/Markdown links for that, Id' leave push out of this. It's perfectly fine to have just periodic (no need to check the push anymore)

Reply:
Okay, I removed the push-based triggering here.